packages/expo-updates/android/src/main/java/expo/modules/updates/db/entity/UpdateEntity.java

```diff
@@ -46,22 +46,28 @@
   @ColumnInfo(name = "launch_asset_id")
   public Long launchAssetId = null;
 
+  @ColumnInfo(name = "manifest")
```

Comment:
Is there a reason why we haven't renamed the attribute as well? It seems like a good policy to have it match the column name.

Reply:
Yeah, I was planning to do that in a later PR to reduce the scope of this one, but wanted to make all the database schema changes in the same PR.

Reply:
Can do the rename here though, if you think it would be clearer.

Reply:
either alternative makes sense to me!

Reply:
Working on this now; going to open a separate PR because it touches a lot of files and I don't want to pollute this one (and also I'm scared of all these comments being deleted if I push another commit to this branch 😂 )